docs/content/recipes/interoperability/identifier-mapping.md

```diff
@@ -0,0 +1,384 @@
+
+While the minting of identifiers is often done in isolation of other organisations,  there are instances of databases who reuse identifiers from a well known community database. For example, the [Human Protein Atlas](https://www.proteinatlas.org/) database reuses [Ensembl](https://www.ensembl.org/) identifiers for their data records. In these cases there is no need to map between the data instances in the two databases, the data is already connected through the common identifier. However, this means that the Human Protein Atlas must ensure that their definition of the concept is exactly aligned with the Ensembl identifier for **all** application uses.
+
+### Identifier Equivalences
+
+Identifier equivalences can come about in several ways and capture different forms of relationship as presented in [Mappings](#Mappings). For example, a database may declare that their record is the same as an entry in another because they share the same name, or they may declare it based on a common representation (gene sequence or InChI). To support reuse of the data, the `provenance` of the cross-references need to be made explicit. 
+
+Depending on the nature of the data, there are different ways that equivalences can be computed. The following is a list of recipes that give details.
+
+
+
+### Exchanging Identifier Mappings
+
+There are several file format for exchanging identifier equivalences. We will discuss the most widely used formats and demonstrate them with a sample of data derived from [ChEMBL](https://www.ebi.ac.uk/chembl/) which provides cross-reference equivalences for two records in the ChEMBL database to UniProt proteins.
+
+#### Text File
+
+The simplest way to exchange equivalences is in a simple text file, which could be structured as a tab-separated-value (TSV) file. Such a file usually consists of two columns, one per dataset, and each row represents an equivalence declaration. The interpretation is that the two identifiers on the same row are equivalent in some way. These files tend to carry little to no metadata about the mappings, i.e. the mechanism by which the mapping was derived is not given, nor are details of the verison of the datasets that were linked.
+
+The following example shows the mapping equivalences between ChEMBL target components (proteins) and UniProt proteins.
+
+```
+ChEMBL_Target_Component    UniProt
+CHEMBL_TC_4803    A0ZX81
+CHEMBL_TC_2584    A1ZA98 
+```
```

Comment:
Is this mean to be a bit of TSV content? then use TABs and use the TSV syntax highlighting.

Reply:
Changed it to tabs, how do you go about using TSV syntax highlighting?

I tried 
```tsv

```

but that didn't change anything